core/src/main/java/nu/yona/server/CacheConfiguration.java

```diff
@@ -0,0 +1,80 @@
+package nu.yona.server;
```

Comment:
We usually put the copyright comment above the `package` line

Reply:
True. I now corrected ``CoreConfiguration`` too.